packages/components/forms/src/text-input/TextInput.tsx

```diff
@@ -22,43 +23,42 @@ export const _TextInput = (
   }: TextInputProps,
   ref: React.Ref<HTMLInputElement>,
 ) => {
-  const [valueState, setValueState] = useState(value);
+  const textInputRef = useRef(null);
```

Comment:
@gui-santos you were right 🙂 (Although I don't know if I do it correctly. probably no)